src/main/resources/react/src/pages/AddGoods/AddGoods.js

```diff
@@ -146,212 +146,143 @@ const AddGoods = () => {
+                </p>
+                <textarea className="description_textarea" />
+              </div>
+              <div className="files">
+                <h3>Загрузите фотографии ваших вещей</h3>
+                <p>Первое фото станет обложкой карточки товара</p>
+                <p>Загружено фотографий {imageFiles.length} из 10</p>
+                <div className="files_wrapper">
+                  {preViewImage.map((url, index) => (
+                    <ImagePhoto
+                      key={index}
+                      url={url}
+                      index={index}
+                      onDragStart={(e) => dragStartHandler(e, index)}
+                      onDragLeave={(e) => dragEndHandler(e)}
+                      onDragEnd={(e) => dragEndHandler(e)}
+                      onDragOver={(e) => dragOverHandler(e)}
+                      onDrop={(e) => dropHandler(e, index)}
+                      removeImage={removeImage}
+                    />
+                  ))}
+
+                  {imageFiles.length < 10 && (
+                    <AddFileInput onChange={filesAddHandler} />
+                  )}
                 </div>
-                <div className="characteristics_item">
-                  <h4>Размер (одежда)</h4>
-                  <CheckBox
-                    text="50 - 80 см"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="80 - 92 см"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="92 - 104 см"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="110 - 122 см"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="128 - 146 см"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="146 - 164 см"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
+              </div>
+              <div className="bottom_block">
+                <div className="buttons_block">
+                  <ButtonAdv type="submit" />
+                  <Button
+                    whatClass="preview"
+                    text="Предпросмотр"
```

Comment:
Are we expecting text for other localizations?

Reply:
I will translate in the future